lib/components/map/base-map.js

```diff
@@ -125,7 +76,8 @@ const mapStateToProps = (state, ownProps) => {
 
 const mapDispatchToProps = (dispatch, ownProps) => {
   return {
-    setLocation: (type, location) => { dispatch(setLocation({ type, location })) }
+    setLocation: (type, location) => { dispatch(setLocation({ type, location })) },
+    updateMapState: (props) => { dispatch(updateMapState(props)) }
```

Comment:
Btw, I would highly recommend matching argument patterns here if you can. For the obvious reason (matching) and also so that you can write a simpler `mapDispatchToProps`. If the argument patterns match then you could write it like:

```js
const mapDispatchToProps = {setLocation, updateMapState}
```